core/src/test/java/hudson/util/AtomicFileWriterTest.java

```diff
@@ -75,6 +76,7 @@ public void setUp() throws IOException {
 
     @Test
     public void symlinkToDirectory() throws Exception {
+        assumeThat(Functions.isWindows(), is(false));
```

Comment:
There is enough of these (3) now that IMO Jenkins should have a test helper to call for "is_symlink_supported(path)".
Given that most windows users use the MSI to install Jenkins and that runs as SYSTEM which they do not change and that means we are not testing functionallity that will be used on a large proportion of windows installs.

Reply:
There is [JENKINS-37862](https://issues.jenkins-ci.org/browse/JENKINS-37862) to move symlink stuff out of core in the first place, but I agree that while it's here it would make sense to test it on Windows where possible.

What do you think is the best way to check if they are supported on Windows? Just try/catch around `Files#createSymbolicLink`? Create a process that runs `cmd.exe /C mklink /d foo bar` and check the return code?